demos/functions/views/demo/workbox-background-sync/sw.hbs

```diff
@@ -0,0 +1,18 @@
+{{{WORKBOX_SW_IMPORT}}}
+
+const bgSyncPlugin = new workbox.backgroundSync.Plugin('myQueueName');
+
+const networkOnlyStrategy = new workbox.strategies.NetworkOnly({
+  plugins: [bgSyncPlugin],
+});
+
+const route = new workbox.routing.RegExpRoute(
+  /^https\:\/\/jsonplaceholder.typicode.com\/posts/,
```

Comment:
Please use a local URL. Just create a file to reference.

Reply:
@gauntface  Will these demos be hosted somewhere?

If not a local file just have extra complications with `sync` event. As the actual network needs to be turned down + the local server needs to be turned down and only then a valid `sync` is registered. 

Thats the reason I chose to be with an external URL altogether

Reply:
Yes - they are hosted on Firebase here: http://workbox-demos.firebaseapp.com/

Reply:
Just to add a 👍 to Matt's request. It would be ideal if we didn't rely on third-party hosted APIs and could just use a local URL here instead.